### ISSUE TITLE
Incorrect Memory Limit description

### ISSUE THREAD

**Issue by tweksteen** [google/grr-doc#38]
The [Memory Limit](https://github.com/google/grr-doc/blob/master/admin.adoc#memory-limit) section of the admin page states:

"Default soft limit is 500MB, but GRR should only use about 30MB. Some volatility plugins can use a lot of memory so we try to be generous. Hard limit is double the soft limit. This is configurable from the config file."

- This appears to be true on Windows, but on Linux there is no hard limit implemented.
- The hard limit is [hard-coded](https://github.com/google/grr/blob/master/client/nanny/windows_nanny.cc#L371) and therefore not configurable.

**Comment by ashishmgupta** [comment]
@tweksteen The link in your above post is broken.

**Comment by grrrrrrrrr** [comment]
Yeah limits have changed over time. This section is clearly from a time where we still used Volatility so around 2013. Do you want to just send a PR to fix it?
